doc/sphinx-guides/source/container/index.rst

```diff
@@ -1,28 +1,12 @@
 Container Guide
 ===============
 
-Running the Dataverse software in containers is quite different than in a :doc:`standard installation <../installation/prep>`.
-
-Both approaches have pros and cons. These days, containers are very often used for development and testing,
-but there is an ever rising move toward running applications in the cloud using container technology.
-
-**NOTE:**
-**As the Institute for Quantitative Social Sciences (IQSS) at Harvard is running a standard, non-containerized installation,
-container support described in this guide is mostly created and maintained by the Dataverse community on a best-effort
-basis.**
-
-This guide is *not* about installation on technology like Docker Swarm, Kubernetes, Rancher or other
-solutions to run containers in production. There is the `Dataverse on K8s project <https://k8s-docs.gdcc.io>`_ for this
-purpose, as mentioned in the :doc:`/developers/containers` section of the Developer Guide.
```

Comment:
Maybe we should keep a variant of this to make it clearer what is out of scope.

Reply:
Ok, so maybe clarify the relationship to k8s, rancher, swarm, etc.